resources/views/user/studentProfile.blade.php

```diff
@@ -0,0 +1,157 @@
+@extends( "layouts.app" )
+
+<!--
+@section( "head" )
+	<meta name="csrf-token" content="{{ csrf_token() }}" />
+@endsection
+-->
+
+@section( "title", "Welcome" )
+
+@section( "content" )
+	<div class="padded content student profile">
+		<h1>Student profile</h1>
+		<div class="courses">
+			<h2>My courses</h2>
+			<div id="courseKeyInputContainer" class="ui action input">
+				<input id="courseKey" type="text" placeholder="Course key">
+				<button id="addCourse" class="ui primary button">Add course</button>
+			</div>
+			<div class="courses table">
+				<table id="currentCourses" class="ui striped table bluemarket-table">
+					<thead class="bluemarket-thead">
+						<tr>
+						<th>Course</th>
+						<th>Professor</th>
+						<th>Schedule</th>
+						</tr>
+					</thead>
+					<tbody>
+						<tr>
+						<td>Computer Graphics</td>
+						<td>Doctor Strange</td>
+						<td>Wed 12:00, Spring 2019</td>
+						</tr>
+						<tr>
+						<td>Web Development</td>
+						<td>Doctor Doom</td>
+						<td>MoThu 14:30, Spring 2019</td>
+						</tr>
+						<td>Software Engineering</td>
+						<td>Captain America</td>
+						<td>TuFri 11:30, Spring 2019</td>
+						</tr>
+					</tbody>
+					</table>
+			</div>
+			<div id="courseFound" class="ui coupled first modal">
+				<div class="header">Add course</div>
+				<div class="content">
+					<p class="newCourseInfo" id="courseName">Advanced Databases</p>
+					<p class="newCourseInfo" id="courseTeacher">Doctor Octopus</p>
+					<p class="newCourseInfo" id="courseSchedule">MoFri 13:00, Spring 2019</p>
+				</div>
+				<div class="actions">
+					<div class="ui cancel button">Cancel</div>
+					<div id="confirmAddCourse" class="ui primary button">Confirm</div>
```

Comment:
No good reason to use a `div` for buttons (bad for a11y). Maybe try an anchor `a` or a plain ol' `button` 👍